Documentation/HowTo/add-syntax-highlighting-with-pygments.md

```diff
@@ -0,0 +1,21 @@
+# How to add syntax highlighting with Pygments
+[Splash](https://github.com/JohnSundell/Splash) and its [official plugin](https://github.com/JohnSundell/SplashPublishPlugin) are great tools for highlighting swift syntax in Publish. 
+
+However, some people write not only in Swift, but also in many other languages. There comes the [SwiftPygmentsPublishPlugin](https://github.com/Ze0nC/SwiftPygmentsPublishPlugin) to help. 
```

Comment:
Suggestion: "There comes the ... to help" -> "That's when ... can be really useful"